tests/curators/test_curators_quickstart_example.py

```diff
@@ -180,3 +252,35 @@ def test_anndata_curator_no_var(small_dataset1_schema: ln.Schema):
     artifact = curator.save_artifact(key="example_datasets/dataset1_no_var.h5ad")
     artifact.delete(permanent=True)
     anndata_schema_no_var.delete()
+
+
+def test_mudata_curator(
+    mudata_papalexi21_subset_schema: ln.Schema, small_dataset1_schema: ln.Schema
+):
+    mudata_schema = mudata_papalexi21_subset_schema
+    mdata = ln.core.datasets.mudata_papalexi21_subset()
+    # TODO: refactor organism
+    bt.settings.organism = "human"
+    # wrong dataset
+    with pytest.raises(InvalidArgument):
+        ln.curators.MuDataCurator(pd.DataFrame(), mudata_schema)
+    # wrong schema
+    with pytest.raises(InvalidArgument):
+        ln.curators.MuDataCurator(mdata, small_dataset1_schema)
+    curator = ln.curators.MuDataCurator(mdata, mudata_schema)
+    assert curator.slots.keys() == {
+        "obs",
+        "rna:obs",
+        "hto:obs",
+        "rna:var",
+    }
+    artifact = curator.save_artifact(key="mudata_papalexi21_subset.h5mu")
+    assert artifact.schema == mudata_schema
+    assert artifact.features.slots.keys() == {
```

Comment:
Where do we test the actual content of the feature sets?

Reply:
Adding values uses he same code used in DataFrameCurator and AnnDataCurator

Reply:
Ok! ☺️